src/expr.rs

```diff
@@ -1721,9 +1730,13 @@ fn rewrite_call_inner<R>(context: &RewriteContext,
     let fmt = ListFormatting {
         tactic: tactic,
         separator: ",",
-        trailing_separator: match context.config.fn_call_style {
-            IndentStyle::Visual => SeparatorTactic::Never,
-            IndentStyle::Block => context.config.trailing_comma,
+        trailing_separator: if force_no_trailing_comma {
```

Comment:
could get rid of the nested match by writing `if force_no_trailing_comma || context.config.fn_call_style == IndentStyle::Visual`